exporter/awsxrayexporter/internal/translator/segment.go

```diff
@@ -37,6 +37,9 @@ const (
 const (
 	awsLocalService  = "aws.local.service"
 	awsRemoteService = "aws.remote.service"
+	// EKS/K8s attributes generated for X-Ray K8s Spans/Segments
+	awsEKSCluster   = "EKS.Cluster"
+	awsK8sNamespace = "K8s.Namespace"
```

Comment:
We should be following kubernetes semantic conventions here for attribute names. https://opentelemetry.io/docs/specs/otel/resource/semantic_conventions/k8s/#cluster

Reply:
AWS wants to present its own attribute view in traces and metrics. that's the tricky part we can't directly take the attribute name from OTel semconv.

Reply:
On the output side that's correct.  On the input side shouldn't the attributes align with the semantic conventions?

Reply:
Is it correct on the output side?
I think, I would be confused as a developer if the attributes morph from the collection to the rendering

Reply:
General question here. Are attributes cases sensitive? Does `K8s.Namespace` conflict at all `k8s.namespace.name`?

Reply:
> On the output side that's correct. On the input side shouldn't the attributes align with the semantic conventions?

Since the same changes need to go to multiple outputs(xray and emf), so the changes are made in the aws processor(in-house).

Reply:
> Is it correct on the output side? I think, I would be confused as a developer if the attributes morph from the collection to the rendering

I think that's why we're trying to use our own attribute name definitions. and make sure what we sent is what we want to present in UI.

Reply:
> General question here. Are attributes cases sensitive? Does `K8s.Namespace` conflict at all `k8s.namespace.name`?

AWS uses Camel naming style for presenting the data. `k8s.namespace.name` generated from instrumentation was renamed by aws in-house processor into `K8s.Namespace`, and then fan out to all AWS outputs.

Reply:
Thanks all for the comments, I want to call out - these attributes will only be generated/available on users who enabled AWS APM components. There is no impact to regular OTel or AWS Distro OTel users. They will not see any of these attributes in the telemetry.

Reply:
This isn't true though. Any user could add these attributes manually with a processor in the pipeline. You do not control the data being consumed by this exporter at all times. Not relying on semantic conventions and behavior associated with them may (unlikely) cause a user to see unintended behavior in the back end right? 

What happens if a user manually adds these resource attributes to spans? At the very least we should define and document the behavior associated with these.